### PR TITLE
chore: Update README.md

### PR DESCRIPTION
**Pull request by marcoscaceres**
Update README to reflect joint deliverable status.

### CODE REVIEW

**Review by reillyeon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,22 +1,7 @@
...
 
-This is the repository in which the W3C DeviceOrientation Event specification is maintained.
-
-The latest version of this specification can be found at [https://www.w3.org/TR/orientation-event/](https://www.w3.org/TR/orientation-event/)
-
-## Useful links
-
-* [W3C DAS WG home page](https://www.w3.org/das/)
-* [W3C DAS WG mailing list](https://lists.w3.org/Archives/Public/public-device-apis/)
+This is the repository in which the W3C's [Device Orientation and Motion](https://www.w3.org/TR/orientation-event/) specification, which is jointly maintained [Device and Sensors Working Group](https://www.w3.org/groups/wg/das/) and the [Web Applications Working Group](https://www.w3.org/groups/wg/webapps/).
```
Comment (anssiko) [inline README.md:3]:
```suggestion
This is the repository in which the W3C's [Device Orientation and Motion](https://www.w3.org/TR/orientation-event/) specification is jointly maintained by the [Devices and Sensors Working Group](https://www.w3.org/groups/wg/das/) and the [Web Applications Working Group](https://www.w3.org/groups/wg/webapps/).
```

**Review by anssiko — CHANGES_REQUESTED** [review]
Some grammar nits. I wish we had a readme template for spec repos to adopt.

**Comment by marcoscaceres** [comment]
@anssiko see the Geolocation one. I think it’s quite good, as it gives a good overview of the API and purpose. We could do something like that here as a follow up.

**Comment by anssiko** [comment]
Geolocation API readme examples are great and could be contributed to MDN developer docs when appropriate to reach more web developers.

I was thinking of a minimal baseline spec repo readme with information on how to contribute, how to build. Something that first into one page. I'm fine with repos having custom readmes, it gives them character.

**Review by anssiko — APPROVED** [review]

**Comment by marcoscaceres** [comment]
> Geolocation API readme examples are great and could be contributed to MDN developer docs when appropriate to reach more web developers.

MDN has a really good [guide](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API/Using_the_Geolocation_API)

I guess if anything, we should probably remove the examples from the Geo README and point people to that 😅 (and maybe we could modernize what's on MDN).

> I was thinking of a minimal baseline spec repo readme with information on how to contribute, how to build. Something that first into one page. I'm fine with repos having custom readmes, it gives them character.

I still think that's better handled by the CONTRIBUTING.md ... For instance, the one for Web Manifest contains how to build instructions, style guide, etc. 

I personally prefer keep the READMEs minimal and without any duplicative information.... more like, if you want to contribute, read CONTRIBUTE.md, if you want to do Y, read Y.MD... and so on.

**Pull request merged**